package.json

```diff
@@ -30,7 +30,8 @@
   "dependencies": {
     "babel-runtime": "5.8.24",
     "fbjs": "^0.7.0",
-    "react-static-container": "^1.0.0-alpha.1"
+    "react-static-container": "^1.0.0-alpha.1",
+    "babel-relay-plugin": "0.7.1"
```

Comment:
Just guessing here, but should it maybe be in devDependencies in that case? It's needed to run the unit tests, but if you're using it in your own project, you should add it to your dependencies (to use it in your transform step).